src/components/guide/guide.css

```diff
@@ -80,43 +87,50 @@
 }
 
 .guideSpacer {
-    width: .3em;
+    width: 0.3em;
     flex-shrink: 0;
 }
 
-.channelPrograms, .timeslotHeadersInner {
+.channelPrograms,
+ .timeslotHeadersInner {
     width: 1800vw;
 }
 
 @media all and (min-width: 37.5em) {
-
-    .channelPrograms, .timeslotHeadersInner {
+    .channelPrograms,
+ .timeslotHeadersInner {
```

Comment:
and these indents are also odd (I won't be adding more comments about these, but all cases I see are following a `@media` line)